src/Subsystems/Intake.h

```diff
@@ -23,18 +23,25 @@ class Intake: public Subsystem {
 private:
 	// It's desirable that everything possible is private except
 	// for methods that implement subsystem capabilities
-	// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
+
 	std::shared_ptr<DoubleSolenoid> actuator;
 	std::shared_ptr<CANTalon> leftSpinner;
 	std::shared_ptr<CANTalon> rightSpinner;
 
-    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
+
 public:
+
+	enum ActuatorValue {kAcuatorOpen = DoubleSolenoid::kForward, kActuatorClosed = DoubleSolenoid::kReverse };
+
 	Intake();
 	void InitDefaultCommand();
-	// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTANTS
 
-    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTANTS
+	//...........Actuator..........
+
+	bool IsActuatorClosed();
```

Comment:
Use 

`bool IsActuatorClosed() const;`

I won't comment on these any longer. Find all getters and make them `const`.